docs/UserGuide.md

```diff
@@ -450,27 +450,27 @@ Example:
 
     Deletes the country note at index 1 of the country notes list panel.
 
-### Getting suggestions on clients: `suggest`
+### Getting suggestions on clients: `client suggest`
 
 Obtains a list of clients based on the suggestion type(s) passed in. This command will also reset the left display panel to its default view.
```

Comment:
do we wanna change this to client suggestion type?

Reply:
ok sure changed